libr/core/cmd_anal.c

```diff
@@ -1007,15 +1052,14 @@ static void list_vars(RCore *core, RAnalFunction *fcn, int type, const char *nam
 	if (name && *name) {
 		var = r_anal_function_get_var_byname (fcn, name);
 		if (var) {
-			r_cons_printf ("%10s  ", var->name);
-			var_accesses_list (fcn, var, access_type);
+			var_accesses_list (fcn, var, access_type, var->name);
 		}
 	} else {
 		r_list_foreach (list, iter, var) {
-			r_cons_printf ("%10s  ", var->name);
-			var_accesses_list (fcn, var, access_type);
+			var_accesses_list (fcn, var, access_type, var->name);
 		}
 	}
+	r_core_seek (core, oaddr, 0);
```

Comment:
this should be placed in `if (type == '=') {` otherwise it won't be executed and you'll change the seek